frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -492,24 +492,25 @@ describe("scenarios > question > custom columns", () => {
     cy.button("Done").should("not.be.disabled");
   });
 
-  describe.skip("contentedtable field (metabase#15734)", () => {
+  describe("ExpressionEditorTextfield", () => {
     beforeEach(() => {
       // This is the default screen size but we need it explicitly set for this test because of the resize later on
       cy.viewport(1280, 800);
 
       openOrdersTable({ mode: "notebook" });
       cy.findByText("Custom column").click();
+
+      cy.get("[contenteditable='true']").as("formula");
+
       popover().within(() => {
-        cy.get("[contenteditable='true']")
-          .as("formula")
-          .type("1+1")
-          .blur();
+        _typeUsingGet("[contenteditable='true']", "1 + 1", 400);
```

Comment:
It was `1+1` in the original repro intentionally. We should keep it like that here.

Btw, I am experimenting with this locally and seeing good results even without helper functions.
Do you mind if I open a PR agains this one?

Reply:
Stress testing regular type (even without a delay) x10 gives me all green:
![image](https://user-images.githubusercontent.com/31325167/118979612-ed193b80-b978-11eb-8fc1-ba419ea6fada.png)

I'll do that in GitHub CI as well, just to make sure everything's working as expected.